docs/DeveloperGuide.md

```diff
@@ -411,17 +424,16 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
       Use case ends.
 
 
-**Use case: Filtering the list of all residents by specific fields**
+**Use case: UC 6. Filtering the list of all residents by specific fields**
 
 **MSS**
 1. User wants to see a list of residents that fall under a certain category.
-2. User uses the filter command to filter the list of residents.
-3. RC4HDB applies the filter.
-4. RC4HDB shows the filtered list.
+2. User uses filters the list of residents. 
```

Comment:
Sounds a bit weird here. Maybe "**User makes a request to RC4HDB for the list of residents to be filtered.**"

Reply:
Alternatively, "User specifies the requirements for filtering". But this might not be appropriate as well, as we will be exposing some of the UI.

Reply:
My suggestion is "User requests for a filtered list from RC4HDB based on the relevant categories".

Reply:
I'm going with nic's suggestion :)